src/context/providers/HubbardContextProvider.js

```diff
@@ -0,0 +1,83 @@
+import { Application } from "@exabyte-io/ade.js";
+import {
+    ApplicationContextMixin,
+    JSONSchemaFormDataProvider,
+    MaterialContextMixin,
+} from "@exabyte-io/code.js/dist/context";
+import { Made } from "@exabyte-io/made.js";
+import { mix } from "mixwith";
+
+const defaultHubbardConfig = {
+    atomicSpecies: "",
+    atomicOrbital: "2p",
+    hubbardUValue: 0.01,
+};
+
+export class HubbardContextProvider extends mix(JSONSchemaFormDataProvider).with(
+    ApplicationContextMixin,
```

Comment:
We're not using application information here, so `ApplicationContextMixin` can be removed. Same goes for the static `Application` variable below